tidb-cloud/connected-care-announcement.md

```diff
@@ -0,0 +1,78 @@
+| Technical support                             | -                            | ✔                                             | ✔                                              | ✔                                         |
+| Initial response time                         | -                            | Business hours                                | 7x24                                           | 7x24                                      |
+| Connected: Clinic service                     | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: AI chat in IM                      | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM subscription for TiDB Cloud alerts   | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM subscription for ticket updates | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM interaction for support tickets | -                            | -                                             | -                                              | ✔                                         |
+| Technical account manager                     | -                            | -                                             | -                                              | ✔                                         |
+| Pricing                                       | Free                         | Greater of $29 or 10% of total monthly charges | Greater of $699 or 20% of total monthly charges | Contact sales                             |
+
+> **Note**
+>
+> Customers from all four support plans have access to the [PingCAP support portal](https://tidb.support.pingcap.com/) for service requests. Advanced features such as Technical Account Managers and [Placeholder] are also available.
+
+## Differences between legacy support services and Connected Care support services
+
+The support plans in the new Connected Care services offer you better connectivity, more personalized support, and cost-effective solutions for different customer needs.
+
+- New **Enterprise** and **Premium** plans: connect customers with modern communication tools and advanced AI capabilities through a completely new set of features, such as advanced monitoring service in Clinic, IM subscription for TiDB Cloud alerts, IM subscription for ticket updates, AI chat in IM, and IM interaction for support tickets.
```

Comment:
```suggestion
- New **Enterprise** and **Premium** plans: connect customers with modern communication tools and advanced AI capabilities through a completely new set of features, such as advanced monitoring service in [TiDB Cloud Clinic](tidb-cloud/tidb-cloud-clinic.md), IM subscription for TiDB Cloud alerts, IM subscription for ticket updates, AI chat in IM, and IM interaction for support tickets.
```

Reply:
tidb-cloud/tidb-cloud-clinic.md will be available on Feb 17, 2025, so we are not able to provide the link now.